spec/unit/puppet/provider/archive/curl_spec.rb

```diff
@@ -15,7 +15,8 @@
       [
         'http://home.lan/example.zip',
         '-o',
-        String,
+        # String,
```

Comment:
looks like this can just be removed, not sure how this was working before

Reply:
done.